src/components/views/elements/TagTile.js

```diff
@@ -22,17 +22,36 @@ import sdk from '../../../index';
 import dis from '../../../dispatcher';
 import { isOnlyCtrlOrCmdKeyEvent } from '../../../Keyboard';
 
+import FlairStore from '../../../stores/FlairStore';
+
 export default React.createClass({
     displayName: 'TagTile',
 
     propTypes: {
-        groupProfile: PropTypes.object,
+        tag: PropTypes.string,
     },
 
     contextTypes: {
         matrixClient: React.PropTypes.instanceOf(MatrixClient).isRequired,
     },
 
+    componentWillMount() {
```

Comment:
`getInitialState` is called before `componentWillMount` in the component lifecycle, so belongs above it in the component imho